File: Assignment23/Program5.c
// 5. Write a program which accept string from user and count number
// of white spaces
// Input :   "MarvellouS"
// Output :   0
// Input :   "MarvellouS Infosystems"
// Output :   1
// Input :   "MarvellouS Infosystems by Piyush Manohar Khairnnar"
// Output :   5

#include<stdio.h>

int CountWhite(char *str)
{
    int iCnt = 0;

    while (*str != '\0')
    {
        if (*str == ' ')
        {
            iCnt++;
        }
        str++;
    }
    return iCnt;
}
int main()
{
    char Arr[20];
    int iRet = 0;
    printf("Enter string\n");
    scanf("%[^'\n']s", &Arr);

    iRet = CountWhite(Arr);
    printf("Space are : %d \n", iRet);

    return 0;
}

File: Assignment38/Program5.java
// 5. Write Java program which accept String from user and display below pattern. 

// Input :  Hello 
// Output : H   
// 			H e   
// 			H e l   
// 			H e l l   
// 			H e l l o   
// 			H e l l   
// 			H e l    
// 			H e l   
// 			H e   
// 			H    

import java.util.*;

class Program5
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);
        String str;

        System.out.println("Enter String");
        str = sobj.nextLine();

        Pattern pobj = new Pattern();
        pobj.Pattern(str);
    }
}

class Pattern 
{ 
 	public void Pattern(String str)  
 	{
		int i = 0, j = 0;
    	char[] Arr = str.toCharArray();

		for(i = Arr.length -1;i != -1;i--)
 		{
 			for(j = 0;j < Arr.length - i ;j++)
 			{
	 			System.out.print(Arr[j]);
 			}
 			System.out.println();
 		}
		for(i = 1;i <= Arr.length - 1;i++)
 		{
 			for(j = 0;j < Arr.length - i;j++)
 			{
	 			System.out.print(Arr[j]);
 			}
 			System.out.println();
 		}
 	} 
}
